### DISCUSSION TITLE
window background blur

### DISCUSSION THREAD

**Discussion by GrootCHENN** [flet-dev/flet#1103] [Q&A] [answered]
### Question

hello! how can i make the background blur or gaussian blur, I don't see some related API.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment, marked as answer]
Pre-release version of Flet (`pip install flet --upgrade --pre`) introduces `Container.blur` property. Here is an example:

```python
import flet as ft

def main(page: ft.Page):

    page.add(
        ft.Stack(
            [
                ft.Container(
                    content=ft.Text("Hello"),
                    image_src="https://picsum.photos/100/100",
                    width=100,
                    height=100,
                ),
                ft.Container(
                    width=50,
                    height=50,
                    blur=10,
                    bgcolor="#44CCCC00",
                ),
                ft.Container(
                    width=50,
                    height=50,
                    left=10,
                    top=60,
                    blur=(0, 10),
                ),
                ft.Container(
                    top=10,
                    left=60,
                    blur=ft.Blur(10, 0, ft.BlurTileMode.REPEATED),
                    width=50,
                    height=50,
                    bgcolor="#44CCCCCC",
                    # border_radius=10,
                    border=ft.border.all(2, ft.colors.BLACK),
                ),
            ]
        )
    )

ft.app(main)
```

**Reply by GrootCHENN** [reply]
Awesome! and one more thing, does it possible to build the view like this,
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/112602646/221750056-1698426a-5f35-4031-b38c-d50b540f93e5.png">

I want the window background to be totally transparent, and blur,
`page.window_bgcolor = ft.colors.TRANSPARENT`
`page.bgcolor = ft.colors.TRANSPARENT`
but the window background is something like black? 
and if i modify the page opacity, the evert widget in the page is opacity

**Comment by TobiasHT5** [comment]
@FeodorFitsner I'm already updating my flet 😅😅, ever since I saw transparent windows, blur was gonna be my next request. Glad you've added it. 
I'm an avid user and contributor to kivy and kivymd frameworks but been playing around with flet ever since it's initial launch and I love the progress you've made.
I think I'm now ready to apply flet in my first real world application.
